numpy/core/include/numpy/halffloat.h

```diff
@@ -28,12 +28,40 @@ int npy_half_gt(npy_half h1, npy_half h2);
 int npy_half_eq_nonan(npy_half h1, npy_half h2);
 int npy_half_lt_nonan(npy_half h1, npy_half h2);
 int npy_half_le_nonan(npy_half h1, npy_half h2);
+
 /* Miscellaneous functions */
-int npy_half_iszero(npy_half h);
-int npy_half_isnan(npy_half h);
-int npy_half_isinf(npy_half h);
-int npy_half_isfinite(npy_half h);
-int npy_half_signbit(npy_half h);
+inline int npy_half_iszero(npy_half h) {
```

Comment:
Interesting, we also use `NPY_INLINE` in many places, I guess that is identical in C99, though and we should maybe just do a "style" cleanup and use `inline` everywhere?